MekHQ/src/mekhq/campaign/rating/AbstractUnitRating.java

```diff
@@ -49,6 +36,7 @@
 import mekhq.campaign.personnel.Person;
 import mekhq.campaign.personnel.Skill;
 import mekhq.campaign.unit.Unit;
+import sun.jvm.hotspot.gc.shared.Space;
```

Comment:
Is this required?

Reply:
I have no idea how that got added... interesting...

Reply:
I wonder if that caused the build failures?

Reply:
I've built it without it, so nope. I've reverted the automatic changes from my IDE.